### ISSUE TITLE
Не получается использовать {item_id} в путях для сохранения файлов

### ISSUE THREAD

**Issue by d2boy** [Quantum-Manager/tracker#154]
При создании нового поля (с использование Quantum Manager) при использовании в пути переменной {item_id} - все изображения сохраняются в папку 0, а не в папку с id статьи.

**Comment by dmitriitux** [comment]
с {item_id} есть пока особенность, чтобы использовать надо после создания уже записи вызывать квантум, где используется. Во время создания id не известен, так как не записано ничего в субд. Механизм перетаскивания я не писал еще, что сначала в некий кеш и после сохранения в субд перетаскивать уже в id с субд что присвоило

**Comment by d2boy** [comment]
Да, я предполага, поэтому попробовал и в существующий материал, но тоже безрезультатно.
